views/order_view.py

```
@@ -0,0 +1,118 @@
+        required_fields = ['user_id', 'total_amount', 'status', 'items']
+        for field in required_fields:
+            if field not in order_data:
+                return json({"error": f"缺少必要字段：{field}"}, status=400)
+        
+        # 创建订单
+        order_number = order_service.create_order(order_data)
+        
+        # 返回成功响应
+        return json({"order_number": order_number}, status=201)
+        
+    except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)